app/services/container_dashboard_service.rb

```diff
@@ -160,10 +160,10 @@ def ems_utilization
 
     daily_provider_metrics.each do |metric|
       date = metric.timestamp.strftime("%Y-%m-%d")
-      used_cpu[date] += metric.v_derived_cpu_total_cores_used
-      used_mem[date] += metric.derived_memory_used
-      total_cpu[date] += metric.derived_vm_numvcpus
-      total_mem[date] += metric.derived_memory_available
+      used_cpu[date] += metric.v_derived_cpu_total_cores_used if metric.v_derived_cpu_total_cores_used.present?
```

Comment:
* I think ```present``` is adequate here
* we decided to skip these metrics in a previous PR - https://github.com/ManageIQ/manageiq/pull/7292